base/test/routes/jwt_test.mustache

```diff
@@ -24,14 +24,14 @@ describe('JWT authentication handling', () => {
   });
 
   it('must accept a valid JWT in a query string param', () => {
-    return fetch(url+'?jwt='+token, {
+    return fetch(url+'?token='+token, {
```

Comment:
Yeah "jwt" would work fine too.  I changed it simply because I spent 20 mins brushing up on JWTs, and all terminology referenced "token" as the preferred generic label, and the fact that the variable name is "token", I decided to just change the query param :)